charts/vector/templates/_pod.tpl

```diff
@@ -71,6 +71,8 @@ containers:
         value: "/host/proc"
       - name: SYSFS_ROOT
         value: "/host/sys"
+      - name: VECTOR_LOG
+        value: "{{ .Values.logLevel | default "info" }}"
```

Comment:
This `default` seems duplicative, given a default value is provided in the `values.yaml`.

Reply:
There is no duplication actually just put default value in the helm chart to be more clear for the users.